src/main/java/by/custom/utilcalculator/service/BundleResourcesServant.java

```diff
@@ -28,11 +28,11 @@ public String getString(final String stringToGetFromBundle) {
 
     private void createBundleResources() {
         try {
-            File fileResources = new File("D:\\IdeaProjects\\custom_calculator_bot\\src\\main\\resources");
-            URL[] resourcesFileUrls = {fileResources.toURI().toURL()};
-            ClassLoader loader = new URLClassLoader(resourcesFileUrls);
+            final File fileResources = new File("D:\\IdeaProjects\\custom_calculator_bot\\src\\main\\resources");
```

Comment:
Using an absolute path is a bad practice. Please, use relevant path like.
this topic could help you https://stackoverflow.com/questions/17351043/how-to-get-absolute-path-to-file-in-resources-folder-of-your-project